ansible/roles/test/tasks/bgp_speaker.yml

```diff
@@ -77,32 +77,56 @@
     - {file_name: "config_3.ini", local_ip: '{{vlan_ips[0]}}', port_num: '7000'}
   delegate_to: "{{ptf_host}}"
 
+- set_fact: addr_family='ipv4'
+  when: addr_family is not defined
+
+- set_fact: mux_ip_1="{% if addr_family=='ipv4' %}{{vlan_ips[1]}}{% else %}{{minigraph_portchannel_interfaces[1]['peer_addr']}}{% endif %}"
```

Comment:
Is it possible not to hard code minigraph_portchanel_interfaces[1] here?

Reply:
Please see above comment.